pkg/convert/convert.go

```diff
@@ -154,7 +154,11 @@ func ResourcesToYamlRegions(resources []map[string]interface{}, outputFile strin
 		for scanner.Scan() {
 			text := scanner.Text()
 			if strings.Contains(text, "cloud_regions:") {
-				finalText += fmt.Sprintf("%s\n%s\n", text, newConfigmap)
+				if strings.HasSuffix(newConfigmap, "\n") {
```

Comment:
You could just check if it doesn't have the ending newline and add it instead :thinking:

Reply:
ah true